include/tables/scene_table.h

```diff
@@ -9,114 +9,114 @@
-/* 0x63 */ DEFINE_SCENE(spot20_scene, g_pn_42, SCENE_SPOT20, 17, 0, 0)
-/* 0x64 */ DEFINE_SCENE(ganon_tou_scene, g_pn_43, SCENE_GANON_TOU, 36, 0, 0)
+/* 0x00 */ DEFINE_SCENE(ydan_scene, g_pn_06, SCENE_YDAN, SCENE_DRAW_CONFIG_YDAN, 1, 2)
+/* 0x01 */ DEFINE_SCENE(ddan_scene, g_pn_08, SCENE_DDAN, SCENE_DRAW_CONFIG_DDAN, 1, 3)
+/* 0x02 */ DEFINE_SCENE(bdan_scene, g_pn_07, SCENE_BDAN, SCENE_DRAW_CONFIG_BDAN, 1, 4)
+/* 0x03 */ DEFINE_SCENE(Bmori1_scene, g_pn_01, SCENE_BMORI1, SCENE_DRAW_CONFIG_BMORI1, 2, 5)
+/* 0x04 */ DEFINE_SCENE(HIDAN_scene, g_pn_03, SCENE_HIDAN, SCENE_DRAW_CONFIG_HIDAN, 2, 6)
+/* 0x05 */ DEFINE_SCENE(MIZUsin_scene, g_pn_04, SCENE_MIZUSIN, SCENE_DRAW_CONFIG_MIZUSIN, 1, 7)
+/* 0x06 */ DEFINE_SCENE(jyasinzou_scene, g_pn_05, SCENE_JYASINZOU, SCENE_DRAW_CONFIG_JYASINZOU, 1, 8)
+/* 0x07 */ DEFINE_SCENE(HAKAdan_scene, g_pn_02, SCENE_HAKADAN, SCENE_DRAW_CONFIG_HAKADAN, 2, 9)
+/* 0x08 */ DEFINE_SCENE(HAKAdanCH_scene, g_pn_54, SCENE_HAKADANCH, SCENE_DRAW_CONFIG_HAKADAN, 2, 10)
+/* 0x09 */ DEFINE_SCENE(ice_doukutu_scene, g_pn_10, SCENE_ICE_DOUKUTO, SCENE_DRAW_CONFIG_ICE_DOUKUTO, 0, 0)
+/* 0x0A */ DEFINE_SCENE(ganon_scene, none, SCENE_GANON, SCENE_DRAW_CONFIG_NONE, 2, 0)
+/* 0x0B */ DEFINE_SCENE(men_scene, g_pn_11, SCENE_MEN, SCENE_DRAW_CONFIG_MEN, 0, 0)
+/* 0x0C */ DEFINE_SCENE(gerudoway_scene, g_pn_49, SCENE_GERUDOWAY, SCENE_DRAW_CONFIG_GERUDOWAY, 0, 0)
+/* 0x0D */ DEFINE_SCENE(ganontika_scene, g_pn_09, SCENE_GANONTIKA, SCENE_DRAW_CONFIG_GANONTIKA, 0, 0)
+/* 0x0E */ DEFINE_SCENE(ganon_sonogo_scene, none, SCENE_GANON_SONOGO, SCENE_DRAW_CONFIG_GANON_SONOGO, 0, 0)
+/* 0x0F */ DEFINE_SCENE(ganontikasonogo_scene, none, SCENE_GANONTIKA_SONOGO, SCENE_DRAW_CONFIG_GANONTIKA_SONOGO, 0, 0)
+/* 0x10 */ DEFINE_SCENE(takaraya_scene, g_pn_51, SCENE_TAKARAYA, SCENE_DRAW_CONFIG_NONE, 0, 0)
+/* 0x11 */ DEFINE_SCENE(ydan_boss_scene, none, SCENE_YDAN_BOSS, SCENE_DRAW_CONFIG_YDAN_BOSS, 0, 0)
+/* 0x12 */ DEFINE_SCENE(ddan_boss_scene, none, SCENE_DDAN_BOSS, SCENE_DRAW_CONFIG_NONE, 0, 0)
+/* 0x13 */ DEFINE_SCENE(bdan_boss_scene, none, SCENE_BDAN_BOSS, SCENE_DRAW_CONFIG_BDAN, 0, 0)
+/* 0x14 */ DEFINE_SCENE(moribossroom_scene, none, SCENE_MORIBOSSROOM, SCENE_DRAW_CONFIG_NONE, 1, 0)
+/* 0x15 */ DEFINE_SCENE(FIRE_bs_scene, none, SCENE_FIRE_BS, SCENE_DRAW_CONFIG_HIDAN, 0, 0)
+/* 0x16 */ DEFINE_SCENE(MIZUsin_bs_scene, none, SCENE_MIZUSIN_BS, SCENE_DRAW_CONFIG_MIZUSIN_BS, 0, 0)
+/* 0x17 */ DEFINE_SCENE(jyasinboss_scene, none, SCENE_JYASINBOSS, SCENE_DRAW_CONFIG_NONE, 0, 0)
+/* 0x18 */ DEFINE_SCENE(HAKAdan_bs_scene, none, SCENE_HAKADAN_BS, SCENE_DRAW_CONFIG_HAKADAN, 0, 0)
+/* 0x19 */ DEFINE_SCENE(ganon_boss_scene, none, SCENE_GANON_BOSS, SCENE_DRAW_CONFIG_NONE, 0, 0)
+/* 0x1A */ DEFINE_SCENE(ganon_final_scene, none, SCENE_GANON_FINAL, SCENE_DRAW_CONFIG_GANON_FINAL, 0, 0)
+/* 0x1B */ DEFINE_SCENE(entra_scene, none, SCENE_ENTRA, SCENE_DRAW_CONFIG_NONE, 0, 0)
+/* 0x1C */ DEFINE_SCENE(entra_n_scene, none, SCENE_ENTRA_N, SCENE_DRAW_CONFIG_NONE, 0, 0)
```

Comment:
I view NONE as more of a DEFAULT, cause it still has a little bit of configuration going on. it just sets all of the segments to an empty dlist and such. It also parallels nicely with `sDefaultDisplayList`

Reply:
Yeah I was hesitant too, but `_default` also suggests that what it does would be done in all other configs, which isn't the case,
And afaik no scene relies on the defaults `_none` sets. It's also not universal defaults either, why would an empty dlist be the sensible default instead of null bytes (e.g. a black rgba16 image).
The intent with `_none` seems more "I don't care about this" to me, which to me feels more `_none`y than `_default`y

I've also considered renaming `sDefaultDisplayList` but wasn't sure what to. Maybe `sDrawConfigNoneDList`.
I think the main reason it exists as an array at all is to not copypaste the same thing for opa and xlu

Reply:
Default just means its the one to use when it doesnt need anything special. Its a perfect description of what it does imo.
I dont think its accurate to say that it does nothing. And default doesnt have to mean that every other case needs to do it as well. Its just the case that you default to when you dont need any of the specific ones